pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -515,24 +515,24 @@ func toKubeContainerStatus(status *runtimeapi.ContainerStatus, runtimeName strin
 
 // executePreStopHook runs the pre-stop lifecycle hooks if applicable and returns the duration it takes.
 func (m *kubeGenericRuntimeManager) executePreStopHook(pod *v1.Pod, containerID kubecontainer.ContainerID, containerSpec *v1.Container, gracePeriod int64) int64 {
-	klog.V(3).Infof("Running preStop hook for container %q", containerID.String())
+	klog.V(3).Infof("Running preStop hook for container %q(id=%q) of pod %s", containerSpec.Name, containerID.String(), format.Pod(pod))
```

Comment:
The output of this would be easier to scan if there was a space between the container name and its ID

Reply:
Thanks for the advice, I have changed the code.